homeworks/ihor.zapisochniy_IgorFromUA/4-basics/basics.js

```diff
@@ -0,0 +1,23 @@
+const myBody = document.querySelector('body');
+
+const firstNumber = Number(prompt('enter first number'));
+const secondNumber = Number(prompt('enter second number'));
+
+const addingOperator = firstNumber + secondNumber;
+const subtractionOperator = firstNumber - secondNumber;
+const multiplicationOperator = firstNumber * secondNumber;
+const divisionOperator = firstNumber / secondNumber;
+const remainderOperator = firstNumber % secondNumber;
+const exponentiationOperator = firstNumber ** secondNumber;
+
+myBody.innerHTML = `
+    <h1>You entered <span class="accent">${firstNumber}</span> and <span class="accent">${secondNumber}</span></h1>
```

Comment:
It is considered a best practice to render page content using HTML rather than relying solely on JavaScript. HTML loads faster, allowing users to see some content while the page is loading. Please move the HTML content to the index.html file

Reply:
The title should be **Basic math operation**

Reply:
Get HTML element by `data-attributes`